packages/modules/redis/src/redis-container.ts

```diff
@@ -0,0 +1,72 @@
+import { AbstractStartedContainer, GenericContainer, StartedTestContainer } from "testcontainers";
+
+const REDIS_PORT = 6379;
+
+export class RedisContainer extends GenericContainer {
+  private password? = "";
+  private persistenceVolume? = "";
+
+  constructor(image = "redis:7.2") {
+    super(image);
+  }
+
+  public withPassword(password: string): this {
+    this.password = password;
+    return this;
+  }
+
+  public withPersistence(sourcePath: string): this {
+    this.persistenceVolume = sourcePath;
+    return this;
+  }
+
+  public override async start(): Promise<StartedRedisContainer> {
+    this.withExposedPorts(...(this.hasExposedPorts ? this.exposedPorts : [REDIS_PORT]))
+      .withCommand([
+        "redis-server",
+        ...(this.password ? [`--requirepass "${this.password}"`] : []),
+        ...(this.persistenceVolume ? ["--save 1 1 ", "--appendonly yes"] : []),
+      ])
+      .withStartupTimeout(120_000);
+    if (this.persistenceVolume) this.withBindMounts([{ mode: "rw", source: this.persistenceVolume, target: "/data" }]);
```

Comment:
I don't think this is something we would like to do. Testcontainers encourages to use `copyFilesToContainer` instead and make sure the tests are portable across docker environments.

Reply:
I was thinking about persisting Tests. But on the view of making tests independent, you're right. I will fix it.

Reply:
@djakielski AFAIK this is the only change pending

Reply:
@cristianrgreco I had some trouble with correct way of importing data via redis cli when executing command in the container. I solved it via bash script.